src/tests/Equinor.ProCoSys.Auth.Tests/Caches/PersonCacheTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.Auth.Caches;
+using Equinor.ProCoSys.Auth.Person;
+using Equinor.ProCoSys.Auth.Time;
+using Microsoft.Extensions.Options;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Equinor.ProCoSys.Auth.Tests.Caches
+{
+    [TestClass]
+    public class PersonCacheTests
+    {
+        private PersonCache _dut;
+        private ProCoSysPerson _person;
+        private readonly Guid _currentUserOid = new Guid("{3BFB54C7-91E2-422E-833F-951AD07FE37F}");
+        private Mock<IPersonApiService> _personApiServiceMock;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            TimeService.SetProvider(new ManualTimeProvider(new DateTime(2020, 1, 1, 0, 0, 0, DateTimeKind.Utc)));
+
+            _personApiServiceMock = new Mock<IPersonApiService>();
+            _person = new ProCoSysPerson { FirstName = "Erling", LastName = "Braut Haaland"};
+            _personApiServiceMock.Setup(p => p.TryGetPersonByOidAsync(_currentUserOid)).Returns(Task.FromResult(_person));
+
+            var optionsMock = new Mock<IOptionsMonitor<CacheOptions>>();
+            optionsMock
+                .Setup(x => x.CurrentValue)
+                .Returns(new CacheOptions());
+
+            _dut = new PersonCache(
+                new CacheManager(),
+                _personApiServiceMock.Object,
+                optionsMock.Object);
+        }
+
+        [TestMethod]
+        public async Task GetAsync_ShouldReturnPersonFromPersonApiServiceFirstTime()
+        {
+            // Act
+            var result = await _dut.GetAsync(_currentUserOid);
+
+            // Assert
+            AssertPerson(result);
+            _personApiServiceMock.Verify(p => p.TryGetPersonByOidAsync(_currentUserOid), Times.Once);
```

Comment:
What is the purpose of Verifying this method?

Reply:
Not that important here, but we assert that it has been called once and only once both here and in test below.
In test below we call _dut.GetAsync, but still is TryGetPersonByOidAsync called only once